packages/hoppscotch-common/src/pages/settings.vue

```diff
@@ -106,6 +106,23 @@
           </section>
         </div>
       </div>
+
+      <div
+        class="md:grid md:gap-4 md:grid-cols-3"
+        v-if="platform.ui?.additionalSettingsSections?.length"
+      >
+        <div class="p-8 md:col-span-1">
+          <h3 class="heading">{{ t("settings.additional") }}</h3>
+        </div>
+        <div class="p-8 space-y-8 md:col-span-2">
+          <template
+            v-for="item in platform.ui?.additionalSettingsSections"
+            :key="item.id"
+          >
+            <component :is="item" />
+          </template>
+        </div>
+      </div>
```

Comment:
Instead of this way of implementation, I think for additional settings, we should make the section work same as how Interceptors and Theming has a section in the app. So, on the side we have a header and subheading, followed by the content on the right.